pkg/cmd/server/kubernetes/master/master_config.go

```diff
@@ -492,6 +510,29 @@ func buildKubeApiserverConfig(
 		return nil, err
 	}
 
+	// we don't use this anymore
```

Comment:
I'll probably update this comment to explicitly mention we don't use legacy audit at all. All auditing in origin uses the new advanced feature.

Reply:
done